experimental/packages/opentelemetry-sdk-metrics-base/test/Instruments.test.ts

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import { InstrumentationLibrary } from '@opentelemetry/core';
+import { Resource } from '@opentelemetry/resources';
+import { AggregationTemporality, InstrumentDescriptor, MeterProvider, MetricReader, PointData, PointDataType } from '../src';
+import { TestMetricReader } from './export/TestMetricReader';
+import { assertMetricData, assertPointData, commonValues, commonAttributes, defaultResource, defaultInstrumentationLibrary } from './util';
+import { Histogram } from '../src/aggregator/types';
+import { ObservableResult } from '@opentelemetry/api-metrics-wip';
+
+describe('Instruments', () => {
+  describe('Counter', () => {
+    it('should add common values and attributes', () => {
```

Comment:
no assertions for this test?

Reply:
Yes, I'm testing it that it doesn't throw with any common values and attributes. I'd update the test title to reflect that.